deployment/hr-data-generator/createHRData.sh

```diff
@@ -15,11 +15,11 @@
 #
```

Comment:
I think the directory for this file should be renamed so not to include a reference to `hr-data-generator`

Reply:
I'm happy for this to stay as-is, but change the path that's used here to point to a parent directory `../Palisade-hr-data-generator/....` or whatever it's called, similarly to how `Palisade-services` is treated

Reply:
are you refering to the link in the file?
java -cp $FILE uk.gov.gchq.syntheticdatagenerator.CreateData $@
if so, what should it be?

Reply:
My mistake, just need to update the bottom comment on `Failed to find hr-data-generator jar` to `Failed to find example-runner jar`